src/components/dialog/dialog.component.js

```diff
@@ -49,22 +55,21 @@ class Dialog extends Modal {
     this.window.addEventListener('resize', this.centerDialog);
 
     if (this.props.autoFocus) {
-      return this.focusDialog();
+      this.focusDialog();
     }
 
-    return null;
+    return setFocusTrap(this._innerContent);
```

Comment:
I used return because eslint shout to me that get method need to have return value.
Maybe better will be to leave it as it was in the original version with return null and just before return use setFocusTrap?
```
setFocusTrap(this._innerContent);
return null;
```
What do you think about this?

Reply:
Maybe it would be better for setFocusTrap to return a reference to the element that has focus?

Reply:
@dawidzarzycki Following our discussions I'm OK to leave this function as-is.